src/w2grid.js

```diff
@@ -1896,7 +1897,7 @@
             var hasHiddenSearches = false;
             // add hidden searches
             for (var i = 0; i < this.searches.length; i++) {
-                if (!this.searches[i].hidden || this.searches[i].value == null) continue;
```

Comment:
not sure why you remove value for searches. There is this notion of hidden searches. If you make a search hidden, it will be always submitted with this value, if it is not null